packages/mui-joy/src/Slider/Slider.tsx

```diff
@@ -0,0 +1,508 @@
+import {
+  SliderUnstyled,
+  SliderValueLabelUnstyled,
+  unstable_composeClasses as composeClasses,
+} from '@mui/base';
+import { OverridableComponent } from '@mui/types';
+import { unstable_capitalize as capitalize } from '@mui/utils';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import * as React from 'react';
+import { Theme, useThemeProps } from '../styles';
+import styled from '../styles/styled';
+import shouldSpreadAdditionalProps from '../utils/shouldSpreadAdditionalProps';
+import sliderClasses, { getSliderUtilityClass } from './sliderClasses';
+import { SliderProps, SliderTypeMap } from './SliderProps';
+
+const useUtilityClasses = (ownerState: SliderProps) => {
+  const { disabled, orientation, track, size, color, variant } = ownerState;
+
+  const slots = {
+    root: [
+      'root',
+      disabled && 'disabled',
+      orientation === 'vertical' && 'vertical',
+      track === 'inverted' && 'trackInverted',
+      track === false && 'trackFalse',
+      color && `color${capitalize(color)}`,
+      variant && `variant${capitalize(variant)}`,
+      size && `size${capitalize(size)}`,
+    ],
+    rail: ['rail'],
+    track: ['track'],
+    mark: ['mark'],
+    markActive: ['markActive'],
```

Comment:
```suggestion
```

We are not having non-slot anymore. The valid slots should be:
- root
- rail
- track
- mark
- markLabel
- thumb
- valueLabel